iphone/Classes/TiUtils.h

```diff
@@ -662,11 +662,23 @@ typedef enum {
 + (BOOL)isRetinaiPhone6Plus;
 
 /**
- Whether or not the current device has a 5.8 inch retina display (iPhone X).
- @return _YES_ if the current device has a 5.8 inch retina display, _NO_ otherwise.
+ Whether or not the current device has a 5.8 inch super retina display (iPhone X).
+ @return _YES_ if the current device has a 5.8 inch super retina display, _NO_ otherwise.
  */
 + (BOOL)isRetinaiPhoneX;
```

Comment:
Should this be renamed to "isRetinaiPhoneXorXS"? something along those lines? Seems the Xs and X share the same resolution/density: https://www.cnet.com/news/iphone-xs-specs-vs-iphone-xs-max-xr-and-x-whats-new-and-better/

Reply:
We used to use the same for the `*S` version (like for the iPhone 4 and 6), but we can change this, sure. The only concern I'm having is that some people may use this public method in their native modules, which would break now.